### PR TITLE
Simplify Readme

### LINKED ISSUE #40: Simplify readme
Organize readme so it's easier to read. Experiment with different ideas to improve readability.

### PR DESCRIPTION
**Pull request by GabrielBottari**
Closes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,38 +128,37 @@ On completion, you will see the following message:
...
-### Github Actions
+### Continuous Integration
 
-This templates integrates Github Actions as Continuous Integration. The existing workflows are under `.github/workflows`. 
+This templates integrates Github Actions for its Continuous Integration. The existing workflows are under `.github/workflows`. 
 To have the CI working, you must:
-- Link your repository with [Coveralls](https://coveralls.io/repos/new).
-- (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
+1. Link your repository with [Coveralls](https://coveralls.io/repos/new).
+1. (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
```
Comment (yann510) [inline README.md:161]:
```suggestion
2. (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
```

**Review by yann510 — APPROVED** [review]
Nice stuff!!!!

File: README.md (hunk 1 above)
Comment (GabrielBottari) [reply README.md:161]: Did you know, if you put only 1s, markdown generates the numbers automatically

**Pull request merged**